pandas/core/generic.py

```diff
@@ -5601,53 +5601,98 @@ def clip(self, lower=None, upper=None, axis=None, inplace=False,
         """
         Trim values at input threshold(s).
 
+        Elements above/below the upper/lower thresholds will be changed to
+        upper/lower thresholds. Clipping data is a method for dealing with
+        out-of-range elements. If some elements are too large or too small,
+        clipping is one way to transform the data into a reasonable range.
+
         Parameters
         ----------
-        lower : float or array_like, default None
-        upper : float or array_like, default None
-        axis : int or string axis name, optional
-            Align object with lower and upper along the given axis.
+        lower : float, array-like or None, default None
+            Lower threshold for clipping.  Values smaller than `lower` will be
+            converted to `lower`.
+        upper : float, array-like or None, default None
+            Upper threshold for clipping.  Values larger than `upper` will be
+            converted to `upper`.
+        axis : {0 or 'index', 1 or 'columns', None}, default None
+            Apply clip by index (i.e. by rows) or columns.
         inplace : boolean, default False
             Whether to perform the operation in place on the data
-                .. versionadded:: 0.21.0
+                .. versionadded:: 0.21.0.
+        *args, **kwargs
+            Additional keywords have no effect but might be accepted
+            for compatibility with numpy.
 
         Returns
         -------
-        clipped : Series
+        `Series` or `DataFrame`.
+            Original input with those values above/below the
+            `upper`/`lower` thresholds set to the threshold values.
+
+        See Also
+        --------
+        DataFrame.clip : Trim values at input threshold(s).
+        Series.clip : Trim values at input threshold(s).
+        Series.clip_lower : Return copy of the input with values below given
+            value(s) truncated.
```

Comment:
would just mention Series.clip here (not upper/lower)

Reply:
Why don't you find `clip_upper` and `clip_lower` related? I suggested them, but actually I was wondering if I was missing something, or if it was being considered deprecating them. It seems to me that `clip(upper=X)` could be used for `clip_upper`.

Reply:
going to deprecate lower/upper but that’s not the reason
when showing a DataFrame method only like to show 1 Series method and and any related DataFrame method 

just a preference to not have too many things in See Also

Reply:
Makes sense to me.

But if I'm not wrong, the title is misleading but this docstring is for the Series.clip method too.

Reply:
ahh ok in that case this is ok